test/int/testSetup.ts

```diff
@@ -39,9 +40,12 @@ export function setup(port?: number, launchProps?: any) {
     if (launchProps) {
         testLaunchProps = launchProps;
     }
-    return ts.setup({entryPoint: DEBUG_ADAPTER, type:'chrome', patchLaunchArgs: patchLaunchArgs, port: port});
+    return ts.setup({ entryPoint: DEBUG_ADAPTER, type: 'chrome', patchLaunchArgs: patchLaunchArgs, port: port });
 }
 
-export function teardown() {
-    return ts.teardown();
+export async function teardown() {
+    // TODO: This is a short-term fix because stopping chrome.exe when the debug session ends is not working properly
+    // We need to fix the product/tests to kill chrome properly and remove this code
+    execSync('taskkill /im chrome.exe', { stdio: 'ignore' });
```

Comment:
Does this throw on non-windows or when chrome.exe doesn't exist?

Reply:
It does, nice catch. Surrounding with try/catch...